docs/source/usage/plugins/shadowgraphy.rst

```diff
@@ -0,0 +1,108 @@
+``--shadowgraphy.slicePoint``             Specifies at what ratio of the total depth of the z dimension, the slice for the field extraction should be set.
+                                          The value given should lie between ``0.0`` and ``1.0``.                            
+``--shadowgraphy.focusPos``               Focus position of lens system relative to slicePoint. The focus position is given in SI units. 
+``--shadowgraphy.fileName``               Output file prefix for openPMD output.          
+``--shadowgraphy.ext``                    Backend for openPMD output.
+``--shadowgraphy.fourierOutput``          If enabled, the fields will also be stored on disk in in ``(x, y, \omega)`` Fourier space in an openPMD file.
+========================================= ==============================================================================================================================
+
+.. note::
+   Currently the plugin only supports an integration slice in the xy-plane, which means that for probing setups the probe pulse should propagate in z direction.
+   The moving window can't be activated or deactivated during the plugin integration loop.
+
+
+Output
+^^^^^^
+Plot the first shadowgram that is stored in the simulation output directory ``simOutput``.
+
+.. code:: python
+
+   import matplotlib.pyplot as plt
+   import numpy as np
+   import openpmd_api as io
+
+
+   def load_shadowgram(series):
+      i = series.iterations[[i for i in series.iterations][0]]
+
+      shadowgram_tmp = i.meshes["shadowgram"][io.Mesh_Record_Component.SCALAR].load_chunk()
+      unit = i.meshes["shadowgram"].get_attribute("unitSI")
+      series.flush()
+
+      return shadowgram_tmp * unit
+
+
+   def load_meshgrids(series):
+      i = series.iterations[[i for i in series.iterations][0]]
+
+      xspace_tmp = i.meshes["Spatial positions"]["x"].load_chunk()
+      xunit = i.meshes["Spatial positions"]["x"].get_attribute("unitSI")
+      series.flush()
+      xspace = xspace_tmp * xunit
+
+      yspace_tmp = i.meshes["Spatial positions"]["y"].load_chunk()
+      yunit = i.meshes["Spatial positions"]["y"].get_attribute("unitSI")
+      series.flush()
+      yspace = yspace_tmp * yunit
+
+      return np.meshgrid(xspace, yspace)
+
+
+   path = "/home/carste06/SCRATCH/runs/2024_ShadowgraphyDev/movingwindow11/simOutput"
+
+   series = io.Series(path + "/shadowgraphy_" + "%T." + "bp5", io.Access.read_only)
+   shadowgram = load_shadowgram(series)
+   xm, ym = load_meshgrids(series)
+   series.close()
+
+   fig, ax = plt.subplots(figsize=(10, 10))
+   ax.pcolormesh(xm, ym, shadowgram)
+   ax.set_aspect("equal")
+
+
+Shadowgram Size and Moving Window
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The size of the pixels is the size of the cells in the simulation divided by the resolution in the plugin ``CELL_WIDTH_SI / shadowgraphy::params::xRes`` and ``CELL_HEIGHT_SI / shadowgraphy::params::yRes``.
```

Comment:
Do you subsample? Our do you skip every other cell? (The description explains subsampling)

Reply:
According to this line of code: https://github.com/ComputationalRadiationPhysics/picongpu/pull/4868/files#diff-6f072cdd2bf72a84ca9a6f25d1239e2d382d66c92a59351c2e09dbf871ef58cfR270 you are not subsampling - thus this should be a multiplication and not a division, right?

Reply:
You are right, it's a typo with the division